ome-demoserver.yml

```diff
@@ -83,6 +83,17 @@
         chdir: /opt/omero/server/selfsigned
         creates: server.p12
 
+    - name: Delete broken Python 3 web app configuration
+      become: yes
+      file:
+        path: "{{ omero_web_basedir }}/config/{{ item }}"
+        state: absent
+      with_items:
+        - django-prometheus.omero
+        - gunicorn-config.py
+      notify:
+        - restart omero-web
+
```

Comment:
This was a one off task related to the major breaking upgrade. If it stays in the playbook it's going to delete and recreate these files on every run. There's a record in the commit history, so how about delete it? Or comment it out.